os-eap64-ping/configure.sh

```diff
@@ -20,4 +20,5 @@ sed -i "s/##VERSION##/$VERSION/g" ${DEST}/module.xml
 
 DEST="$JBOSS_HOME/modules/system/layers/openshift/net/oauth/core/main/"
 mkdir -p "$DEST"
-cp -p ${SOURCES_DIR}/oauth-20100527.jar "$DEST"
+cp -p ${ADDED_DIR}/modules/system/layers/openshift/net/oauth/core/main/module.xml \
```

Comment:
Haven't found the corresponding place this was defined earlier, not sure how this relates to the moving of artifact definitions out of image sources and into cct_modules (yet)

Reply:
These were part of the os-eap*-modules module before.  It didn't seem right that one module was creating the eap module and another was populating the jar in it.  Basically, all of the code working with oauth.jar has been consolidated into a single module.  Here's the other relevant change: https://github.com/jboss-openshift/cct_module/pull/213/files#diff-c03b6c0a09c0feccf76969b7f2c0fc6c